synapse/federation/transport/server.py

```diff
@@ -261,10 +261,10 @@ def new_func(request, *args, **kwargs):
             except NoAuthenticationError:
                 origin = None
                 if self.REQUIRE_AUTH:
-                    logger.exception("authenticate_request failed")
+                    logger.warn("authenticate_request failed")
                     raise
             except Exception:
-                logger.exception("authenticate_request failed")
+                logger.warn("authenticate_request failed")
```

Comment:
ditto

Reply:
It gets reraised so should get logged as appropriate by the usual HTTP exception loggers, but will add to differentiate between the cases of "no auth given" vs "there was an error"